README.md

```diff
@@ -0,0 +1,210 @@
+- **Example**: See [the ESLint `no-invalid-this` docs](http://eslint.org/docs/rules/no-invalid-this) for examples of allowed and disallowed behavior.
+
+####[`no-shadow`](http://eslint.org/docs/rules/no-shadow)
+- **Setting**: `"error"`
+- **Explanation**: Do not redefine variables in a child scope that are already declared in the upper scope.
+- **Example**:
+
+        // Linter error
+        var foo = 'bar';
+        if (fizz) {
+            var foo = 'buzz';
+        }
+
+####[`object-curly-spacing`](http://eslint.org/docs/rules/object-curly-spacing)
+- **Setting**: `["error", "never"]`
+- **Explanation**: Do not insert extra spaces inside of curly brackets.
+- **Example**:
+
+        // Correct pattern
+        var obj = {foo: 42}
+
+        // Linter error
+        var obj2 = { foo: 42 }
+
+####[`one-var`](http://eslint.org/docs/rules/one-var)
+- **Setting**: `["error", "always"]`
+- **Explanation**: Only use one variable declaration per function. This reinforces awareness of ES5's block scoping and hoisting, which can reduce bugs.
+- **Example**:
+
+        // Correct pattern
+        function() {
+            var foo, bar, baz,
+                fizz = 'buzz',
+                foz = 49;
+        }
+
+        // Linter error
+        function() {
+            var fizz = 'buzz';
+
+            if (foo) {
+                var baz = 'bar';
+            }
+        }
+
+####[`one-var-declaration-per-line`](http://eslint.org/docs/rules/one-var-declaration-per-line)
+- **Setting**: `["error", "initializations"]`
+- **Explanation**: Variables that are not initialized can be declared on the same line. Only one initialization is allowed per line.
+- **Example**:
+
+        // Correct pattern
+        var foo, bar,
+            fizz = 'buzz';
+
+        // Linter error
+        var foo, bar, fizz = 'buzz';
+
+####[`quote-props`](http://eslint.org/docs/rules/quote-props)
+- **Setting**: `["error", "consistent"]`
+- **Explanation**: If no properties of an object need to be wrapped in quotes (i.e., they are all camelCased strings), then they may remain unwrapped by quotes. If any object property requires quotes, then wrap all object properties in quotes regardless of their individual need for it.
+- **Example**:
+
+        // Correct patterns
+        var foo = {
+            bar: 'buzz',
+            biff: 'bop'
+        }
+
+        var foo2 = {
+            'bar': 'buzz',
+            'uh-oh-kebab-cased': 'bop'
+        }
+
+        // Linter errors
+        var foo = {
+            'bar': 'buzz',
+            'biff': 'bop'
+        }
+
+        var foo2 = {
+            bar: 'buzz',
+            'uh-oh-kebab-cased': 'bop'
+        }
+
+        // Syntax error
+        var foo2 = {
+            bar: 'buzz',
+            uh-oh-kebab-cased: 'bop'
+        }
+
+####[`space-before-function-paren`](http://eslint.org/docs/rules/space-before-function-paren)
+- **Setting**: `["error", "never"]`
```

Comment:
Also worth mentioning that eslint supports the `--fix` option for this rule, meaning it will auto-rewrite the code to whichever way you specify the option, so it's super easy to fix globally once we pick a style.